02/app.js

```diff
@@ -38,3 +36,11 @@ function checkData(e) {
 		console.log('Done');
 	}
 }
+
+function resetLabelColor() {
+	for (const el of formEl.elements) {
+		if (el !== submitEl) {
```

Comment:
Zamiast wyszukiwać przycisk i tutaj wykorzystywać zmienną z zewn. to sprawdziłbym po prostu atrybut `type` np.
`if(el.type !== 'submit')`